QUEST_LV_0200.tsv

```diff
@@ -7028,10 +7028,10 @@ QUEST_LV_0200_20150511_007031	Better stand back as it's dangerous
 QUEST_LV_0200_20150511_007032	Better give up as it's dangerous
 QUEST_LV_0200_20150511_007033	Notice/The magic circle strengthened when you removed the 4 crystals!!#5
 QUEST_LV_0200_20150511_007034	Notice/!/You must gather the spell of monsters in the pot and bury it in front of the magic circle.#5
-QUEST_LV_0200_20150511_007035	Beraking the container/HAMMERING/2/TRACK_BEFORE/None
+QUEST_LV_0200_20150511_007035	Breaking the container/HAMMERING/2/TRACK_BEFORE/None
 QUEST_LV_0200_20150511_007036	Better do it yourself
-QUEST_LV_0200_20150511_007037	Brewer Dorjen is angry that he is stopped from burning the forest. Try to calm Dorjen down.
-QUEST_LV_0200_20150511_007038	If its not the honey they're after, then there's only one thing. Check the Baciu Altar that the monsters flocked to.
+QUEST_LV_0200_20150511_007037	Brewer Dorjen is angry because he is stopped from burning the forest. Try to calm Dorjen down.
```

Comment:
Brewer Dorjen is angry because he **was** stopped from burning the forest. Try to calm Dorjen down.